frontends/election-manager/src/screens/write_ins_screen.tsx

```diff
@@ -155,131 +78,17 @@ export function WriteInsScreen(): JSX.Element {
     );
   }
 
-  const transcriptionContestIds = [
-    ...new Set(temporaryTranscriptions.map((t) => t.contest_id)),
-  ];
-  const contestsWithWritins = election.contests.filter((c) =>
-    transcriptionContestIds.includes(c.id)
-  );
-
-  function sortWithEmptyStringLast(a: string, b: string) {
```

Comment:
FYI I think there is some magic beau was doing in this code where if you adjudicate transcibeOptionA as transcibeOptionB it adjudicates BOTH of those not just the first one, he also might have been doing some sorting magic in here we will need to think about when implementing the wiring up of the adjudication stuff. Although I don't think any of that is mission critical.

Reply:
Restating myself to ensure I understand: if you transcribe some things as “Bob” and some others as “Robert” and then you adjudicate “Bob” as “Robert”, then both “Bob” and “Robert” are adjudicated as “Robert”.